vagrant_support/playbook.yml

```diff
@@ -0,0 +1,176 @@
+      yum: name={{item}} state=present
+      with_items:
+        - libselinux-python
+        - policycoreutils-python
+        - python-urllib3
+        - python-pyasn1
+        - python2-ndg_httpsclient
+        - pyOpenSSL
+        - htop
+
+    - name: permissive selinux
+      selinux: state=permissive policy=targeted
+
+    - name: disable selinux on reboot
+      selinux: state=disabled
+
+    - name: 'add {{WEB_HOSTNAME}} to /etc/hosts'
+      lineinfile:
+        dest: /etc/hosts
+        regexp: '{{WEB_HOSTNAME}}'
+        line: '{{WEB_IP}} {{WEB_HOSTNAME}}'
+        owner: root
+        group: root
+        mode: 0644
+
+    - name: 'add {{DB_HOSTNAME}} to /etc/hosts'
+      lineinfile:
+        dest: /etc/hosts
+        regexp: '{{DB_HOSTNAME}}'
+        line: '{{DB_IP}} {{DB_HOSTNAME}}'
+        owner: root
+        group: root
+        mode: 0644
+
+- hosts: db
+  become: true
+  tasks:
+    - name: fetch ius-release.rpm
+      get_url:
+        url: https://moz-devservices-bmocartons.s3.amazonaws.com/third-party/ius-release.rpm
+        dest: /tmp/ius-release.rpm
+        mode: 0600
+
+    - name: fetch mysql rpm
+      get_url:
+        url: https://moz-devservices-bmocartons.s3.amazonaws.com/third-party/mysql-community-release-el6-5.noarch.rpm
+        dest: /tmp/mysql-community-release-el6-5.noarch.rpm
+        mode: 0600
+
+    - name: add some extra repos
+      yum: name={{item}} state=present
+      with_items:
+        - /tmp/ius-release.rpm
+        - /tmp/mysql-community-release-el6-5.noarch.rpm
+
+    - name: install mysql-server
+      yum: name=mysql-community-server state=present
+
+    - name: copy my.cnf
+      copy:
+        src: my.cnf
+        dest: /etc/my.cnf
+
+    - name: enable mysqld
+      service: name=mysqld enabled=yes
+
+    - name: restart mysqld
+      service: name=mysqld state=restarted
+
+    - name: 'check for bugs_bmo (failure is ok)'
+      shell: mysql -u root -e 'show databases' | grep bugs_bmo
+      register: bugs_bmo
+      ignore_errors: true
+
+    - name: create bugs_bmo
+      shell: |-
+        mysqladmin -u root create bugs_bmo
+        mysql -u root -e 'GRANT ALL ON bugs_bmo.* TO bugs@"{{WEB_IP}}" IDENTIFIED BY "bugs"'
+      when: bugs_bmo|failed
+
+- hosts: web
+  become: true
+  tasks:
+    - name: install web-specific packages
+      yum: name={{item}} state=present
+      with_items:
+        - autoconf
+        - automake
+        - binutils
+        - bison
+        - byacc
+        - elfutils
+        - flex
+        - gcc
+        - gcc-c++
+        - gd
+        - gd-devel
+        - gettext
+        - git
+        - libtool
+        - make
+        - mod_perl
+        - mod_perl-devel
+        - openssl
+        - openssl-devel
+        - patch
+        - patchutils
+        - perl
+        - perl-core
+        - pkgconfig
+        - rpm-build
+        - graphviz
+
+    - name: fetch cpanm
+      get_url:
+        url: http://cpanmin.us
```

Comment:
Is it at all possible for us to vendor tarball ```cpanm```, so that we're not doing the equivalent of ```curl | bash```?